src/descriptors/power_system_inputs.json

```diff
@@ -4,6 +4,15 @@
             "name": "simulation",
             "description": "Simulation name"
         },
+        {
```

Comment:
These definitions are very tailored to the CDM data format. I am not sure if we could say this file is general anymore.